### DISCUSSION TITLE
FR: Add a setting to change volume of all applications when scrolling on the general icon

### DISCUSSION THREAD

**Discussion by Yvand** [File-New-Project/EarTrumpet#639] [Ideas]
I use an external USB sound card to output sound to a SPDIF interface.
A side effect is that changing general volume doesn't have any effect (only muting works), I can only control volume on individual applications.
If I could bind the general volume to all the applications, it would fix this issue, and the buttons to control volmume on my keyboard will be useful again!

**Comment by Eridyne** [comment]
I strongly agree with this, albeit for other reasons - it's sometimes necessary to be able to boost the volume of a single application over all other audio, and currently there's no easy way to do that.
A modifier key or toggle to change all application volumes would work reasonably for this, as one could drop all application volumes, then adjust a single application upwards again.
I believe this would be best together with #196 and #329 - but would be valuable alone, regardless.
